Stats/Supporting Files/da.lproj/Localizable.strings

```diff
@@ -2,11 +2,11 @@
 //  Localizable.strings
 //  Stats
 //
-//  Created by Casper Sørensen (casperes1996) on 18/11/2021.
+//  Created by Serhiy Mytrovtsiy on 27/08/2020.
 //  Using Swift 5.0.
-//  Running on macOS 12.1.
+//  Running on macOS 10.15.
```

Comment:
Sorry, you can change this part. Leave only copyright (line 9) in the original form.

Reply:
Ah, I see what you mean :) - updated